source/Mission.cpp

```diff
@@ -307,6 +331,8 @@ void Mission::Load(const DataNode &node)
 		displayName = name;
 	if(hasPriority && location == LANDING)
 		node.PrintTrace("Warning: \"priority\" tag has no effect on \"landing\" missions:");
+	if(!outfitStr.empty() && !outfitterStr.empty())
+		node.PrintTrace("Warning: both outfit and outfitter set this will cause unpredictable behavior:");
```

Comment:
I mean its not really unpredictible, right?

```suggestion
		node.PrintTrace("Warning: both outfit and outfitter being defined is not supported, and will cause both to use the probabilities defined last:");
```